BluePic-Server/Sources/Server/RouteUtils.swift

```diff
@@ -277,7 +269,7 @@ func updateImageJSON(json: JSON, withRequest request: RouterRequest) throws -> J
  - returns: NSError object
  */
 func generateInternalError() -> NSError {
-  return NSError(domain: BluePic.Domain, code: BluePic.Error.Internal.rawValue, userInfo: [NSLocalizedDescriptionKey: String(BluePic.Error.Internal)])
+  return NSError(domain: BluePic.Domain, code: BluePic.Error.Internal.rawValue, userInfo: [NSLocalizedDescriptionKey: String(describing: BluePic.Error.Internal)])
```

Comment:
`LocalizedError` has no accessible initializers. Is there another type of Error that would be appropriate here? @rfdickerson ?